eng/build.sh

```diff
@@ -297,6 +297,12 @@ if [[ "$restore" == true && "$source_build" != true ]]; then
   dotnet tool restore
 fi
 
+if [[ "$restore" == true ]]; then
+  # Install any dotnet workloads specified in global.json
+  # TODO: Call Install-DotNetWorkloads.ps1 instead
+  dotnet workload install macos
```

Comment:
Assuming everything else looks OK, I will change this to either invoke `Install-DotNetWorkloads.ps1` or will port said script to bash.

Reply:
This is in-progress.